en/sensor_bus/i2c.md

```diff
@@ -32,7 +32,7 @@ For example, you can see/search for this driver in the [px4_fmu-v4_default](http
 To find I2C driver examples, search for **i2c.h** in [/src/drivers/](https://github.com/PX4/PX4-Autopilot/tree/master/src/drivers).
 
 Just a few examples are:
-* [drivers/sf1xx](https://github.com/PX4/PX4-Autopilot/tree/master/src/drivers/distance_sensor/sf1xx) - I2C Driver for [Lightware SF1XX LIDAR](../sensor/sfxx_lidar.md).
+* [drivers/sf1xx](https://github.com/PX4/PX4-Autopilot/tree/master/src/drivers/distance_sensor/sf1xx) - I2C Driver for [Lightware SF1XX LIDAR](../sensor/sfxx_lidar.md). 
```

Comment:
@bkueng This driver seems to have disappeared? Is there a replacement you'd suggest instead?

Reply:
They got renamed for clarity: https://github.com/PX4/PX4-Autopilot/tree/master/src/drivers/distance_sensor/lightware_laser_i2c and https://github.com/PX4/PX4-Autopilot/tree/master/src/drivers/distance_sensor/lightware_laser_serial